docs/make.jl

```diff
@@ -0,0 +1,14 @@
+using Pkg
+Pkg.activate("..")
+Pkg.add("Documenter")
+
```

Comment:
This was not needed :wink: 
```suggestion
```

Reply:
If I remove those lines, I get

julia --project make.jl
ERROR: LoadError: ArgumentError: Package AgentEpidemics not found in current path.
- Run `import Pkg; Pkg.add("AgentEpidemics")` to install the AgentEpidemics package.
Stacktrace:
 [1] macro expansion
   @ ./loading.jl:2208 [inlined]
 [2] macro expansion
   @ ./lock.jl:273 [inlined]
 [3] __require(into::Module, mod::Symbol)
   @ Base ./loading.jl:2183
 [4] #invoke_in_world#3
   @ ./essentials.jl:1088 [inlined]
 [5] invoke_in_world
   @ ./essentials.jl:1085 [inlined]
 [6] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:2176
in expression starting at /home/arindam/AgentEpidemics/docs/make.jl:1

Reply:
Building the documentation on CI will work though.